### DISCUSSION TITLE
Cannot import the RepositoryMining class

### DISCUSSION THREAD

**Discussion by MoonGirl99** [ishepard/pydriller#258] [Q&A]
I am trying to write a script for mining repository commits, but I have a serious problem my script does not import the RepositoryMining class from PyDriller. 

from pydriller import RepositoryMining
    for commit in RepositoryMining(args.url, only_modifications_with_file_types=['.java'], only_branch='master').traverse_commits():
        # Check if the commit was made before or on 2022-12-31
        if commit.committer_date.date() <= datetime(2022, 12, 31).date():
            # Your code here to process the commit as needed
            print(f'Commit hash: {commit.hash}')
            print(f'Commit message: {commit.msg}')
            print(f'Commit date: {commit.committer_date}')

Error : Cannot find reference 'RepositoryMining' in '__init__.py'

** I have installed pydriller and my python version is 3.10 and I am working in Pycharm IDE

**Comment by ishepard** [comment]
the class is now called Repository: https://pydriller.readthedocs.io/en/latest/
